src/core/io/src/4C_io_vtu_reader.hpp

```diff
@@ -0,0 +1,42 @@
+// This file is part of 4C multiphysics licensed under the
+// GNU Lesser General Public License v3.0 or later.
+//
+// See the LICENSE.md file in the top-level for license information.
+//
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#ifndef FOUR_C_IO_VTU_READER_HPP
+#define FOUR_C_IO_VTU_READER_HPP
+
+#include "4C_config.hpp"
+
+#include "4C_io_mesh.hpp"
+
+#include <filesystem>
+#include <string>
+#include <vector>
+
+FOUR_C_NAMESPACE_OPEN
+
+namespace Core::IO::VTU
+{
+  /*!
+   * @brief Read a vtu-mesh into the 4C intermediate representation
+   *
+   * The vtu-format does not have a natural representation of cell- and point-sets. To define these
+   * sets, we assume the following point-/cell-data arrays to be present:
+   *
+   *  - point-arrays "point_set_##": Integer arrays telling whether a point is part of a point-set.
```

Comment:
```suggestion
   *  - zero or more point-arrays named "point_set_##": Integer arrays telling whether a point is part of a point-set.
```

Reply:
Should the "point set" be called "node set"? We seem to use it later with that name.